package.json

```diff
@@ -26,39 +26,47 @@
 		"polyfill"
 	],
 	"dependencies": {
-		"assert": "^2.0.0",
+		"assert": "^2.1.0",
 		"browserify-zlib": "^0.2.0",
 		"buffer": "^6.0.3",
 		"console-browserify": "^1.2.0",
 		"constants-browserify": "^1.0.0",
 		"crypto-browserify": "^3.12.0",
 		"domain-browser": "^4.22.0",
 		"events": "^3.3.0",
-		"filter-obj": "^2.0.2",
 		"https-browserify": "^1.0.0",
 		"os-browserify": "^0.3.0",
 		"path-browserify": "^1.0.1",
 		"process": "^0.11.10",
-		"punycode": "^2.1.1",
+		"punycode": "^2.3.0",
 		"querystring-es3": "^0.2.1",
-		"readable-stream": "^4.0.0",
+		"readable-stream": "^4.4.2",
 		"stream-browserify": "^3.0.0",
 		"stream-http": "^3.2.0",
 		"string_decoder": "^1.3.0",
 		"timers-browserify": "^2.0.12",
 		"tty-browserify": "^0.0.1",
-		"type-fest": "^2.14.0",
-		"url": "^0.11.0",
-		"util": "^0.12.4",
+		"type-fest": "^4.4.0",
+		"url": "^0.11.3",
+		"util": "^0.12.5",
 		"vm-browserify": "^1.1.2"
 	},
 	"devDependencies": {
-		"ava": "^3.15.0",
+		"ava": "^5.3.1",
 		"p-webpack": "^1.0.1",
-		"webpack": "^5.73.0",
-		"xo": "^0.39.1"
+		"webpack": "^5.88.2",
+		"xo": "^0.56.0"
 	},
 	"peerDependencies": {
 		"webpack": ">=5"
+	},
+	"xo": {
+		"rules": {
+			"unicorn/prefer-module": "off",
+			"comma-dangle": "off"
```

Comment:
Can we remove this?

Reply:
xo changed the defaults, so if you want to remove this, you need to update all your files, because they had the previous default.

Reply:
I do intend to do that

Reply:
Done! ✅